src/core/metrics.cc

```diff
@@ -272,6 +272,34 @@ void impl::remove_registration(const metric_id& id) {
     }
 }
 
+void
+impl::remove_metric_replica_family(const seastar::sstring& name, int destination) const {
+    auto entry = _value_map.find(name);
+
+    if (entry == _value_map.end()) {
+        return;
+    }
+
+    for (const auto& metric_instance: entry->second) {
+        const auto& registered_metric = metric_instance.second;
+        remove_metric_replica(registered_metric->get_id(), destination);
```

Comment:
destination is always the same here, `get_local_impl(destination)` inside `remove_metric_replica` could be lifted from the loop, at which point it might be worth just inlining it.